queries.sql

```diff
@@ -1,10 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * FROM animals WHERE name LIKE '%mon';
-SELECT * FROM animals WHERE EXTRACT(YEAR FROM date_of_birth) BETWEEN 2016 AND 2019;
-SELECT * FROM animals WHERE neutered = TRUE AND escape_attempts < 3;
-SELECT * FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
-SELECT * FROM animals WHERE weight_kg > 10.5;
-SELECT * FROM animals WHERE neutered = TRUE;
-SELECT * FROM animals WHERE NOT (name = 'Gabumon');
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT name FROM animals WHERE name LIKE '%mon';
+SELECT date_of_birth FROM animals WHERE EXTRACT(YEAR FROM date_of_birth) BETWEEN 2016 AND 2019;
+SELECT neutered, escape_attempts FROM animals WHERE neutered = TRUE AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
+SELECT weight_kg, escape_attempts FROM animals WHERE weight_kg > 10.5;
```

Comment:
- [ ] Let us kindly replace `weight_kg` to `name` as we can see in the screenshot below
![Screenshot from 2023-05-02 08-13-14](https://user-images.githubusercontent.com/46388113/235613076-48ae6b0b-c5e4-4d2d-af0b-8091a178e5da.png)